### PR TITLE
Fix wp landing screen brush image stretching issue on iPad

### LINKED ISSUE #19484: [Revamp Landing Screen] In WordPress app the brush stroke image appears stretched on Tablets
### Expected behavior
The background brush stroke on the new Landing Screen in the WordPress app is rendered sharp.

### Actual behavior
On the new Landing Screen the brush stroke background image appears stretched.

### Steps to reproduce the behavior
1. Open the WordPress app on an iPad
2. Enable the `newWordPressLandingScreen` feature flag
   1. Log in
   2. Go to `Me` → `AppSettings` → `🐞 Debug`
   3. Enable `newWordPressLandingScreen`
3. Log out
4. Re-open the WordPress App
5. **Expect** the **NEW** landing screen to show
6. **Notice** The brush stroke background image looks stretched.

| iPad | Android Tablet |
| - | - |
|  |  |

P.S. The grayish background color was also added to the Android screen, the issue is only about the stretched brush stroke image that is more visible when opening the full size iPad screenshot (it's also noticeable on iPads).

### PR DESCRIPTION
**Pull request by salimbraksa**
Fixes #19484

This PR fixes an issue where the brush stroke background image on the WordPress Landing Screen appears stretched.

## Screenshots
| Before | After |
| ---- | --- |
|  | iPad Pro - Light |

| iPhone 14 Pro | iPhone SE |
| -------------- | ---------- |
| |  |

## To test:
1. Open the WordPress app on an iPad
2. Enable the newWordPressLandingScreen feature flag
   - Log in
   -  Go to Me → AppSettings → 🐞 Debug
   - Enable newWordPressLandingScreen
3. Log out
4. Re-open the WordPress App
5. **Expect** the NEW landing screen to show
6. **Expect** The brush stroke background image should look like the attached screenshots or the [design](https://github.com/wordpress-mobile/WordPress-iOS/pull/19485#issuecomment-1284308455).

## Regression Notes
1. Potential unintended areas of impact
None.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
None.

3. What automated tests I added (or what prevented me from doing so)
None.

## PR submission checklist:
- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by twstokes** [comment]
@shaunandrews could we please have your input on these visual changes? 🙇

**Comment by shaunandrews** [comment]
The size of the brush stroke looks much better, but can we keep it positioned relative to the logo and text?

CleanShot 2022-10-19 at 12 56 09@2x

File: WordPress/Resources/AppImages.xcassets/Prologue/SplashPrologue/splashBrushStroke.imageset/Contents.json [hunk 1]
```diff
@@ -1,26 +1,16 @@
...
-    {
-      "filename" : "splashBrushStroke@2x.png",
-      "idiom" : "universal",
-      "scale" : "2x"
-    },
-    {
-      "filename" : "splashBrushStroke@3x.png",
-      "idiom" : "universal",
-      "scale" : "3x"
+      "filename" : "splashBrushStroke.pdf",
```
Comment (twstokes) [inline WordPress/Resources/AppImages.xcassets/Prologue/SplashPrologue/splashBrushStroke.imageset/Contents.json:4]: Everything's better with a vector! Thanks for updating this.

**Review by twstokes — APPROVED** [review]
This looks great @salimbraksa and is way better than the previous style. 👏 

The only non-blocking observation I have is that if we felt the need to take this further and account for the vertical height, the vertical offsets could stay more consistent across devices and orientations.

| Portrait | Landscape |
| - | - |
| Simulator Screen Shot - iPad (9th generation) - 2022-10-20 at 11 11 49 | Simulator Screen Shot - iPad (9th generation) - 2022-10-20 at 11 11 54 |

**Pull request merged**